include/minisketch.h

```diff
@@ -77,15 +77,17 @@ void minisketch_add_uint64(minisketch* sketch, uint64_t element);
  * the set elements.
  *
  * If the capacity of `other_sketch` is lower than `sketch`'s, its capacity will be
- * reduce to that of `other_sketch`.
- *
- * It is also possible to perform this operation directly on the serializations,
- * by performing an XOR on its bits.
+ * reduced to that of `other_sketch`.
  *
  * This function is a no-op if the two input sketches do not have the same element
- * size or implementation.
+ * size or implementation. In that case, the function returns 0. Otherwise, i.e., if
+ * merging was performed, it returns the new capactiy of `size` (which is at least 1).
```

Comment:
Wait, that's wrong. The API let's you create sketch with capacity 0.

Reply:
We're using -1 ssize_t as a error indicator elsewhere too. Perhaps that can be done here as well?